app/src/main/res/values-sw360dp/auto_dimens.xml

```diff
@@ -14,4 +14,10 @@
     <dimen name="dp_0.5">1dp</dimen>
     <dimen name="dp_62">62dp</dimen>
     <dimen name="dp_20">20dp</dimen>
+    <dimen name="sp_15">15sp</dimen>
+    <dimen name="dp_45">45dp</dimen>
+    <dimen name="dp_100">100dp</dimen>
+    <dimen name="dp_50">50dp</dimen>
+    <dimen name="dp_60">60dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_60` appears to be unused
 Unused resources make applications larger and slow down builds.